### PR TITLE
Google login dev

### LINKED ISSUE #88: As a developer I want to create a button in sign in page for google auth
linking #17 

AC:
1.When click on the google login it should create a user in our database

### PR DESCRIPTION
**Pull request by Dahlak76**
## Description

Adds the ability for a user to choose one-click google sign in
## Related Issue

closes #88 

# Acceptance Criteria

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

### After

image

## Testing Steps / QA Criteria

log in using google. if your token is already saved, open dev tools, open application and delete the token there and go back to localhost:3000 (or wherever is opened)/signin and you will see the google button

### CODE REVIEW

**Review by nuriyealp — APPROVED** [review]
Great Job!

File: src/Pages/SignIn/Google.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        const googleScript = document.getElementById('google-script')
+        document.body.removeChild(googleScript)
+    }
+ 
+  
+
+  render() {
+    return (
+      <div>
+        <div>click here to sign in with google!!!!!</div>
```
Comment (XeniyaDob) [inline src/Pages/SignIn/Google.js:52]: Let's do a button here instead of text.

**Review by XeniyaDob — APPROVED** [review]
Great job! But we need a button for Sign in with google

File: src/Pages/SignIn/Google.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+class Google extends React.Component {
+
+  constructor(props) {
+    super(props)
+    window.customCallback = this.customCallback.bind(this)
+    this.customCallback=this.customCallback.bind(this);
+  }
+  
+  customCallback(response) {
+    localStorage.setItem("token", response.credential)
```
Comment (nuriyealp) [inline src/Pages/SignIn/Google.js:28]: We need to update token name to myKitchenAppToken.

**Review by XeniyaDob — CHANGES_REQUESTED** [review]
The Google component should be a functional component. Could you please change "Click here to sign in with google" to a button?

File: src/Pages/SignIn/Google.js (hunk 1 above)
Comment (Dahlak76) [reply src/Pages/SignIn/Google.js:52]: ok. sounds good. I'm about to fix the merge conflict now and can do it. 
should I do ... ??
<div>
   <button> Sign In with Google!</button>
</div>

File: src/Pages/SignIn/Google.js (hunk 2 above)
Comment (Dahlak76) [reply src/Pages/SignIn/Google.js:28]: Ok I'll change that to be 
   localStorage.setItem("myKitchenAppTolen", response.credential)

File: src/Pages/SignIn/Google.js (hunk 1 above)
Comment (Dahlak76) [reply src/Pages/SignIn/Google.js]: all done. Messaging to resolve for merge

**Review by Dahlak76 — COMMENTED** [review]
All done and passing all tests

**Review by XeniyaDob — APPROVED** [review]
I am able to sign in with the google button.

**Pull request merged**